ledger/acctupdates.go

```diff
@@ -68,6 +68,15 @@ const baseResourcesPendingAccountsBufferSize = 100000
 // is being flushed into the main base resources cache.
 const baseResourcesPendingAccountsWarnThreshold = 85000
 
+// baseBoxesPendingBufferSize defines the size of the base boxes pending buffer size.
+// At the beginning of a new round, the entries from this buffer are being flushed into the base boxes map.
+const baseBoxesPendingBufferSize = 5000
```

Comment:
@algoidurovic Can the inline comment be expanded (or a new one added) to summarize _why_ we chose the configured value?

I realize the value is liable to change prior to merge.  Consider the note a visual reminder for when we're ready to finalize the value.